src/rust/lib/prelude/src/lib.rs

```diff
@@ -38,6 +38,11 @@ pub use ifmt::*;
 pub use itertools::Itertools;
 pub use lazy_static::lazy_static;
 pub use num::Num;
+pub use maplit::btreemap;
+pub use maplit::btreeset;
+pub use maplit::convert_args;
+pub use maplit::hashmap;
+pub use maplit::hashset;
```

Comment:
I would not expose in prelude: `btreemap`, `btreeset`, and `convert_args`. The first 2 are so rarely used that they could be imported when needed. The `convert_args` name is way too generic to be exported in prelude.